addon/ajax-request.js

```diff
@@ -160,6 +160,15 @@ export default class AjaxRequest {
   }
 
   /**
+   * calls `request()` but forces `options.type` to `DELETE`
+   * alias for `del()`
+   * @public
+   */
+  delete(...args) {
+    return this.del(...args);
```

Comment:
@sandstrom 

You can apply the spread operator to the function's implicit `arguments` variable, so there's no need to put it in the function signature. I think that's what @alexlafroscia is getting at. 😄 
```
delete() {
  return this.del(...arguments);
}
```